src/zope/__init__.py

```diff
@@ -1 +1 @@
-__import__('pkg_resources').declare_namespace(__name__)
+__import__('pkg_resources').declare_namespace(__name__) # pragma: no cover
```

Comment:
> I'm guessing it gets skipped entirely by the namespace machinery

Right, as far as imports go.

> why does coverage pick it up at all?

Because the report phase is based on the files it finds on disk, I'm assuming. It's the same for any module that's not imported at all but is still on disk.